.github/workflows/github-actions.yml

```diff
@@ -0,0 +1,24 @@
+name: Lint and Tests
+on:
+  pull_request:
+    types: [opened, synchronize]
```

Comment:
Add here the `dispatch_workflow` too, please

Reply:
Added dispatch_workflow.